java11/README.md

```diff
@@ -287,33 +294,158 @@ ResolveRequest personRequest = fcClient
+              .tag(Tag.builder().key("gender").value("something").build())
+              .build();
+// Sending Request
+CompletableFuture<TagsResponse> tagsResponseCompletableFuture =
+          fcClient.tagsCreate(tagsRequest);
+
+tagsResponseCompletableFuture.thenAccept(
+          tagsResponse -> {
+            System.out.println(tagsResponse.getRecordId());
+            System.out.println(tagsResponse.getPartnerId());
+            System.out.println(tagsResponse.getTags());
+          });
+```
+
+#### Get Tags
+This will return all customer tags that are associated to a mapped record using `recordId`.
+
+```java
+CompletableFuture<TagsResponse> tagsResponseCompletableFuture = fcClient.tagsGet("recordId");
+
+tagsResponseCompletableFuture.thenAccept(
+          tagsResponse -> {
+            System.out.println(tagsResponse.getRecordId());
+            System.out.println(tagsResponse.getPartnerId());
+            System.out.println(tagsResponse.getTags());
+          });
+```
+
+#### Delete Tags
+This will remove specific or all customer tags that are attached to a mapped record.
+
+```java
+// Building Request
+TagsRequest tagsRequest =
+          FullContact.buildTagsRequest()
+              .recordId("k2")
+              .tag(Tag.builder().key("gender").value("something").build())
+              .build();
+// Sending Request
+CompletableFuture<TagsResponse> tagsResponseCompletableFuture =
+          fcClient.tagsDelete(tagsRequest);
+
+tagsResponseCompletableFuture.thenAccept(
+          tagsResponse -> {
+            System.out.println(tagsResponse.statusCode);
+          });
+```
+
+### Audience
+- `audience.create`
+- `audience.download`
+
+This endpoint can be used in order to obtain multiple individuals based upon the key, value 
+tag inputs (both are required as input) in order to suppress or take action upon certain audiences 
+for data onboarding or audience analysis.
+
+#### Audience Create
+The Audience Creation endpoint requires a atleast one `Tag` and valid `webhookURL` to be present in order to 
+send a message when the audience creation is complete and ready to be downloaded.
+
+```java
+AudienceRequest audienceRequest =
+          FullContact.buildAudienceRequest()
+              .tag(Tag.builder().key("gender").value("male").build())
+              .webhookUrl("your-webhook-url")
+              .build();
+
+ CompletableFuture<AudienceResponse> audienceResponseCompletableFuture =
+          fcClient.audienceCreate(audienceRequest);
+
+ audienceResponseCompletableFuture.thenAccept(
+          audienceResponse -> {
+            System.out.println(audienceResponse.isSuccessful);
+            System.out.println(audienceResponse.getRequestId());
+            });
+```
+
+#### Audience Download
+When `audience.create` result is ready, `requestId` from its response can be used to download the audience data.
+A utility method is provided `getFileFromBytes(String fileName)` which generates a file
+with audience data bytes.
```

Comment:
Would you add that the file is gzipped json file.

Reply:
Done